modules/GPTQ_loader.py

```diff
@@ -33,16 +33,16 @@ def noop(*args, **kwargs):
     for name in exclude_layers:
         if name in layers:
             del layers[name]
-    make_quant(model, layers, wbits, groupsize, faster=faster_kernel, kernel_switch_threshold=kernel_switch_threshold)
+    make_quant(model, layers, wbits, groupsize)
```

Comment:
We could conditionalize this instead
```python
import inspect
[...]
    gptq_args = inspect.getfullargspec(make_quant).args

    make_quant_kwargs = {
        'module': model, 
        'names': layers,
        'bits': wbits,
    }
    if 'groupsize' in gptq_args:
        make_quant_kwargs['groupsize'] = groupsize
    if 'faster' in gptq_args:
        make_quant_kwargs['faster'] = faster_kernel
    if 'kernel_switch_threshold' in gptq_args:
        make_quant_kwargs['kernel_switch_threshold'] = kernel_switch_threshold
    
    make_quant(**make_quant_kwargs)
```
I think that also makes this backwards-compatible with the pre-groupsize GPTQ-for-LLaMA.
The same thing can be repeated for `load_quant` below.

Reply:
Very nice! I'll use your code.

Reply:
I can make a proper pull request shortly, just had to wait on a quantization for triton first for some testing. It does seem to work fine on GPTQ-for-LLaMA old CUDA/new CUDA/triton branches without having to tweak anything.

Reply:
I would appreciate a PR. I'm refactoring several things in the interface and haven't been testing GPTQ much.